frontend/src/components/Header/components/AuthButtons/index.tsx

```diff
@@ -64,8 +76,9 @@ function LoggedInButtons({ name, email, logout }: { name?: string; email?: strin
     </>
   );
 
-  function Content({ logout }: { logout: any }) {
+  function Content() {
```

Comment:
since this function is within the scope of `logout`, we don't need to pass `logout` to the function